### ISSUE TITLE
0.48.12 is not compatible with Mint package manager

### ISSUE THREAD

**Issue by valerio-castelli** [nicklockwood/SwiftFormat#1021]
It looks like the 0.48.12 release has broken compatibility with Mint package manager. The offending commit is [99cf5837c97963d516863d689669500b93fd63e2](https://github.com/nicklockwood/SwiftFormat/commit/99cf5837c97963d516863d689669500b93fd63e2), which removes `LinuxMain.swift` from the `Tests` folder. Unfortunately, this breaks the package installation with Mint:
```
🌱 Finding latest version of SwiftFormat
🌱 Cloning SwiftFormat 0.48.12
🌱 Resolving package
🌱 Building package
error: missing LinuxMain.swift file in the Tests directory
🌱 Encountered error during "swift build -c release". Use --verbose to see full output
🌱  Failed to build SwiftFormat 0.48.12 with SPM
```

We have a few CI pipelines that are currently broken because of this. We can probably pin 0.48.11 for the time being, but it would be better to have a proper fix. 🙏

**Comment by nicklockwood** [comment]
Thanks - I'll restore the Linux.main file.

**Comment by nicklockwood** [comment]
@valerio-castelli can you verify that it now works?

**Comment by nicklockwood** [comment]
Fixed in 0.48.13 (probably)

**Comment by valerio-castelli** [comment]
I confirm that version 0.48.13 fixes the issue, thanks! 🙌
